src/test/com/codingnomads/betty/logic/services/ProbabilityToWinServiceTests.java

```diff
@@ -0,0 +1,52 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.TeamProbabilityToWin;
+import com.codingnomads.betty.logic.models.TeamSentimentScore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProbabilityToWinServiceTests {
+
+    @Test(expected = NullPointerException.class)
+    public void ifTeamSentimentScoreIsNull_throwANullPointerException() {
+        TeamSentimentScore teamSentimentScore = new TeamSentimentScore();
+        teamSentimentScore.setScore(null);
+
+        ProbabilityToWinService service = new ProbabilityToWinService();
```

Comment:
Use the `setUp() `method of the test classes to initialize the service that is going to be used during all the tests.

Reply:
![tenor-195196366](https://user-images.githubusercontent.com/32600398/56577620-ff8be400-65fd-11e9-9207-baca98a26ff4.gif)